docs/custom-transforms-and-renditions.md

```diff
@@ -34,52 +35,32 @@ the repository.
 
 For the ACS repository to talk to a T-Engine, it must know the engine's 
```

Comment:
ACS repository => Repository

Reply:
The text in docs-alfresco was change to be just Repository, so I'm trying to reflect that here too.

Reply:
done